### PR TITLE
removing the very incorrect empty data object from /send_message

### PR DESCRIPTION
**Pull request by nietaki**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -791,7 +791,6 @@ Authorization: Mainframe-Bot <secret>
   "conversation_id": "<conversation ID>",
   "subscription_id": "<subscription ID>",
   "message": "<text message>",
-  "data": {}
```
Comment (howleysv) [inline source/index.html.md]: `{...}`?

Comment (nietaki) [inline source/index.html.md]: I think those are supposed to be working examples, I don't think `{...}` is useful in an example

Comment (howleysv) [inline source/index.html.md]: I just saw it used above in another example

Comment (nietaki) [inline source/index.html.md]: Actually I having a more "full" example for `/post` would be useful as well.

**Pull request merged**